docs/source/tooling/truffle.rst

```diff
@@ -1,59 +1,166 @@
-MythX Security Analysis plugin for Truffle
-==========================================
+.. _tooling.truffle:
 
-MythX Security Analysis plugin for Truffle adds automated smart contract security analysis to the `Truffle framework <https://truffleframework.com>`_.
+Truffle
+=======
 
-Prerequisites
-^^^^^^^^^^^^^
+**MythX for Truffle** is a `Truffle plugin <https://truffleframework.com/docs/truffle/getting-started/writing-external-scripts#third-party-plugin-commands>`_ that adds automated smart contract security analysis to the `Truffle framework <https://truffleframework.com>`_.
 
-The MythX plugin is compatible with Truffle 5.0 or higher. Note that your Truffle project must compile
+The MythX plugin requires Truffle 5.0 or higher. Note that your Truffle project must compile
 successfully for the security analysis to work.
 
-Quickstart
-^^^^^^^^^^
+Installation
```

Comment:
This is a duplicate label to another entry in the docs. We should give it a more explicit name like "Truffle Plugin Installation"

Reply:
That's weird. There's no reason why a Sphinx project can't have the same word as a title on any page it wants.

Turns out that's because there's an optional Sphinx extension: https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html

Going to suggest that we remove that. It doesn't give us anything. If we want labels, we can add them.